src/package-downloader.imjoy.html

```diff
@@ -0,0 +1,32 @@
+<config lang="json">
+{
+  "name": "BioImageIO-Downloader",
+  "type": "web-python",
+  "tags": [],
+  "ui": "",
+  "version": "0.1.0",
+  "api_version": "0.1.7",
+  "description": "Export a BioImage.IO model package from a model RDF",
+  "icon": "https://raw.githubusercontent.com/bioimage-io/bioimage-io-models/main/assets/download-icon.png",
+  "inputs": null,
+  "outputs": null,
+  "env": "",
+  "requirements": [],
+  "dependencies": [],
+  "defaults": {"w": 20, "h": 10},
+  "runnable": true
+}
+</config>
+
+<script lang="python">
+from imjoy import api
+
+class ImJoyPlugin():
+    def setup(self):
+        api.log('initialized')
+
+    def run(self, ctx):
+        api.alert('hello world.')
+
+api.export(ImJoyPlugin())
+</script>
```

Comment:
this hello world example does not run. I'm getting:

```
Uncaught (in promise) Error: Traceback (most recent call last):
  File "/lib/python3.8/site-packages/pyodide/_base.py", line 70, in eval_code
    eval(compile(mod, "<exec>", mode="exec"), ns, ns)
  File "<exec>", line 3, in <module>
  File "/lib/python3.8/site-packages/werkzeug/local.py", line 347, in __getattr__
    return getattr(self._get_current_object(), name)
  File "/lib/python3.8/site-packages/imjoy_rpc/__init__.py", line 46, in __getattr__
    setup_connection(_rpc_context, connection_type, logger)
  File "/lib/python3.8/site-packages/imjoy_rpc/utils.py", line 370, in setup_connection
    manager = PyodideConnectionManager(_rpc_context)
  File "/lib/python3.8/site-packages/imjoy_rpc/connection/pyodide_connection.py", line 35, in __init__
    loop = asyncio.get_event_loop()
  File "/lib/python3.8/asyncio/events.py", line 748, in get_event_loop
    return get_event_loop_policy().get_event_loop()
  File "/lib/python3.8/asyncio/events.py", line 636, in get_event_loop
    self.set_event_loop(self.new_event_loop())
  File "/lib/python3.8/asyncio/events.py", line 656, in new_event_loop
    return self._loop_factory()
  File "/lib/python3.8/asyncio/unix_events.py", line 54, in __init__
    super().__init__(selector)
  File "/lib/python3.8/asyncio/selector_events.py", line 61, in __init__
    self._make_self_pipe()
  File "/lib/python3.8/asyncio/selector_events.py", line 108, in _make_self_pipe
    self._ssock, self._csock = socket.socketpair()
  File "/lib/python3.8/socket.py", line 597, in socketpair
    lsock.listen()
OSError: [Errno 95] Not supported

    at _hiwire_throw_error (pyodide.asm.js:8)
    at _pythonexc2js (pyodide.asm.wasm:0x108433)
    at __runPython (pyodide.asm.wasm:0x109536)
    at Object.Module.__runPython (pyodide.asm.js:8)
    at Object.Module._runPythonInternal (pyodide.asm.js:8)
    at Object.Module.runPython (pyodide.asm.js:8)
    at eval (pyodide.webworker.js:39)
    
```
    
I suppose py3.8 is not supported yet? @oeway how do I specify a python version? through "env"? "requirements"? Couldn't find anything in the docs https://imjoy.io/docs/#/development?id=web-python 
Is there a working example somewhere?

Reply:
Nope, that's the old version of imjoy-core which breaks due to changes in pyodide. It didn't work because the CI somehow breaks and it failed to publish recent changes in the imjoy-core repo. I will fix that.

Reply:
It now passes after a manual re-run:  https://github.com/imjoy-team/imjoy-core/runs/2989520176?check_suite_focus=true

I will fix the reset

Reply:
@oeway , once ready, could you point me to a working example, please?

Reply:
Already done, reload your ImJoy and make sure you have v0.11.32. I am also shipping a development feature for the bioimage.io site directly. Will tag you for the usage in a few minutes.

Reply:
OK, done!

See here: https://github.com/bioimage-io/bioimage.io/pull/137

I will merge it soon

@FynnBe